lib/workflows/send_hv_email.py

```diff
@@ -0,0 +1,175 @@
+from typing import List, Optional, Union
+
+from email_api.emailer import Emailer
+from openstack_query import ServerQuery, UserQuery
+
+from enums.cloud_domains import CloudDomains
+from enums.query.props import ServerProperties
+from enums.query.query_presets import QueryPresetsGeneric
+
+from structs.email.email_params import EmailParams
+from structs.email.email_template_details import EmailTemplateDetails
+from structs.email.smtp_account import SMTPAccount
+
+
+def find_servers_on_hv(
```

Comment:
We're doing this a lot - for all our emailing notifications - maybe we should make a new folder to hold and reuse common query patterns like this in the future